src/sonic-py-common/sonic_py_common/syslogger.py

```diff
@@ -35,6 +42,111 @@ def __init__(self, log_identifier=None, log_facility=DEFAULT_LOG_FACILITY, log_l
         self.logger.addHandler(handler)
 
         self.set_min_log_priority(log_level)
+        
+        if enable_runtime_config:
+            self.register_for_runtime_config(log_identifier)
+            
+    def register_for_runtime_config(self, log_identifier):
+        """Register current log instance to log registry. If DB entry exists, 
+        update log level according to DB configuration; otherwise, save current
+        log level to DB.
+
+        Args:
+            log_identifier (str): key of LOGGER table
+        """
+        if log_identifier not in SysLogger.log_registry:
+            SysLogger.log_registry[log_identifier] = [self]
+        else:
+            SysLogger.log_registry[log_identifier].append(self)
+            
+        from swsscommon import swsscommon
```

Comment:
@Junchao-Mellanox there is only one SysLogger instance per daemon/process. Why we need more than one identifier?

Reply:
Fixed